### ISSUE TITLE
Can't change input modes with keyboard from Percussion Panel

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32803]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

When focused on the drum pad pressing N  or your assigned shortcut for input by duration seem to have no effect and give no speech feedback.

STEPS:
1.  Create a score with a drum kit and navigate to that staff.  Press tab several times to get to the drum pad.
2. Press your assigned shortcut for input by duration mode (Note: there is no default shortcut so you need to have previously assigned one). Notice there is no speech feedback.
3. Press Esc to jump back to the score and press the shortcut again to check whether it had previously changed.  You hear "live live".
4. Press it again several times while on the score and note the usual wording is "input by duration mode" and "normal mode" as it toggles off.
I am unsure whether this is not changing or just not speaking.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7 beta

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by sshadid-code** [comment]
Hello, I’m a college student looking for an open source GitHub issue to complete as part of a course assignment and would like to be assigned this issue. I believe my C++ experience should have me prepared for this task and your extensive setup and contribution guidelines will help the debugging process greatly. I plan to follow your contribution guidelines and submit a PR that meets your QA standards within 2-3 weeks. 

Please let me know if you’re willing to assign me this issue at your earliest convenience as I need to report back to my class once I’ve found an issue to work on. Thank you.

**Comment by jmwmusic** [comment]
@sshadid-code My guess is that this one might need some further diagnosis and design from the development team as I couldn't fully work out what the issue was with the accessibility.  If you are new to the accessibility area we have a priority list of small issues that a large group of blind users using this software would love someone to pick one of.  See the list at [MuseScore Top 10 Small Accessibility Issues](https://groups.io/g/MuseScoreAccessibility/wiki/39926/223910) or maybe look at[25571, #20039, #26867.  you can contact me via the accessibility group linked above if you want more info on using NVDA and how we navigate.

**Comment by sshadid-code** [comment]
@jmwmusic I see, thank you for letting me know. From looking at the Top 10 Small Issues and the three issues you listed, issue [#26867 ](https://github.com/musescore/MuseScore/issues/26867) seems like an interesting bug to tackle that may better fit my current skill level (I have no prior experience with accessibility debugging but am eager to learn). Would it be possible to assign me to [#26867 ](https://github.com/musescore/MuseScore/issues/26867)? 

Also, please let me know if any of my actions/methods of communication are unprofessional or not up to your team's standards. This is my first time contributing to an open source project like this so I'm still getting the hang of things. I'll be sure to reach out to you via the group for NVDA help need be. Thank you for your help.

> [@sshadid-code](https://github.com/sshadid-code) My guess is that this one might need some further diagnosis and design from the development team as I couldn't fully work out what the issue was with the accessibility. If you are new to the accessibility area we have a priority list of small issues that a large group of blind users using this software would love someone to pick one of. See the list at [MuseScore Top 10 Small Accessibility Issues](https://groups.io/g/MuseScoreAccessibility/wiki/39926/223910) or maybe look at[25571, [#20039](https://github.com/musescore/MuseScore/issues/20039), [#26867](https://github.com/musescore/MuseScore/issues/26867). you can contact me via the accessibility group linked above if you want more info on using NVDA and how we navigate.

**Comment by jmwmusic** [comment]
@sshadid-code I'm not part of the development team, just a user and run the support group, so follow whatever the standard steps are to see if you can get assigned to that one.  To me it looks like you don't need any ability with screen reader to work on it as it is a straight keyboard navigation problem.  Notice also the couple of potentially related issues, if you fix one it just might lead to a fix on 2 others!

**Comment by avvvvve** [comment]
@sshadid-code If you want to work on #26867, please leave a comment there, and then I can assign it to you (Github doesn't let me do so until you've left a comment).
